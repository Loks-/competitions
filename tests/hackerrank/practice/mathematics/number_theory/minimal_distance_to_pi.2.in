450214 458009
