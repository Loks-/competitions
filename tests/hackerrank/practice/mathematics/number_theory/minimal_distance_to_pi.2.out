1438862/458004
